builds/development/css/audience.css

```diff
@@ -3268,129 +3287,129 @@ u {
 }
 
 @media (max-width: 767px) {
-  .top .logo a {
-    margin: 0 auto;
-    width: 50%;
+  .top .logo a img {
+    width: 100%;
   }
 }
 
-.top .department {
-  color: #fff;
-  font-family: "Helvetica Neue", "Helvetica", "Arial", sans-serif;
-  letter-spacing: 1px;
+.top .logo .svgLinkContainer {
+  display: block;
+  position: relative;
+  height: 1px;
+  width: 100%;
+  padding-bottom: 29.95839%;
+  z-index: 2;
 }
 
-.top .department > a {
-  display: inline-block;
-  padding: 2px 10px;
-  line-height: 1.2em;
+.top .logo .svgLinkContainer::after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
 }
 
-@media (min-width: 768px) {
-  .top .department > a {
-    border-left: solid 1px #c28e0e;
-  }
+.top .logo .svgContainer {
+  display: block;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: -1;
+  padding-right: 0px;
 }
 
 @media (max-width: 767px) {
-  .top .department > a {
-    border: none;
+  .top .logo .svgLinkContainer {
+    margin: 0 auto;
+    width: 50%;
+    padding-bottom: 14.9792%;
   }
 }
 
-.top .department > a:link {
-  color: inherit;
-  font-weight: 100;
-  text-decoration: none;
+.top .department {
+  color: #fff;
+  font-family: "Archivo Narrow", "Arial Narrow", "Arial", sans-serif;
+  letter-spacing: 0;
+  padding-left: 15px;
+  padding-right: 15px;
 }
 
-.top .department > a:visited {
-  color: inherit;
-  font-weight: 100;
-  text-decoration: none;
+.top .department > a {
+  display: inline-block;
+  line-height: 1.1em;
+  border: none;
+  padding: 0;
 }
 
-.top .department > a:hover {
-  color: inherit;
-  font-weight: 100;
-  text-decoration: none;
+@media (min-width: 992px) and (max-width: 1199px) {
+  .top .department > a {
+    display: inline-block;
+    padding: 2px 0px;
+    line-height: 1.1em;
+  }
 }
 
-.top .department > a:active {
+.top .department > a:link, .top .department a:visited, .top .department a:hover, .top .department a:active {
   color: inherit;
-  font-weight: 100;
+  font-weight: 600;
   text-decoration: none;
 }
 
 .top .department .tagline {
   display: block;
   clear: left;
   color: inherit;
-  font-size: 0.52em;
-  line-height: 1.3em;
-}
-
-.top .department .tagline a:link {
-  color: inherit;
-  font-weight: 400;
-  text-decoration: none;
-}
-
-.top .department .tagline a:visited {
-  color: inherit;
-  font-weight: 400;
-  text-decoration: none;
-}
-
-.top .department .tagline a:hover {
-  color: inherit;
-  font-weight: 400;
-  text-decoration: none;
+  font-family: "Archivo Narrow", "Arial Narrow", "Arial", sans-serif;
+  font-size: 0.8em;
+  line-height: 1.2em;
+  letter-spacing: .5px;
+  text-transform: uppercase;
 }
 
-.top .department .tagline a:active {
+.top .department .tagline a:link, .top .department .tagline a:visited, .top .department .tagline a:hover, .top .department .tagline a:active {
   color: inherit;
-  font-weight: 400;
+  font-weight: 100;
   text-decoration: none;
 }
 
```

Comment:
Another focus style for keyboard navigation:
```
.top .department .tagline a:focus{
    outline:2px dotted #fff;
    outline-offset:1px;
}
```
![site tagline on focus example](https://user-images.githubusercontent.com/6598084/36607358-a414a486-1883-11e8-9c4c-232b54cf4579.png)


@jengarza @bholaday Thoughts?